### PR TITLE
Headless.adoc, net-motors.adoc  update

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
* correct grammar indefinite article
* add Raspberry word

### CODE REVIEW

**Review by aallan — APPROVED** [review]

File: documentation/asciidoc/computers/configuration/headless.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 == Setting up a Headless Raspberry Pi
 
-If you do not use a monitor or keyboard to run your Pi (known as headless), but you still need to do some wireless setup, there is a facility to enable wireless networking and SSH when creating a image.
+If you do not use a monitor or keyboard to run your Raspberry Pi (known as headless), but you still need to do some wireless setup, there is a facility to enable wireless networking and SSH when creating an image.
 
 Once an image is created on an SD card, by inserting it into a card reader on a Linux or Windows machines the xref:configuration.adoc#the-boot-folder[boot folder] can be accessed. Adding certain files to this folder will activate certain setup features on the first boot of the Raspberry Pi.
```
Comment (lurch) [inline documentation/asciidoc/computers/configuration/headless.adoc:5]: @aallan Perhaps it's worth adding some wording here about the built-in support for this in RPi-Imager, now that it's exposed without requiring a special keypress?

Comment (RuffaloLavoisier) [reply documentation/asciidoc/computers/configuration/headless.adoc:5]: how about would like to add a content related to the Raspberry Pi Imager to the technical document. and add a separate guideline for the Raspberry Pi Imager?

**Comment by RuffaloLavoisier** [comment]
@aallan 
I missed it. thanks

**Comment by lurch** [comment]
@aallan Just to clarify, in https://www.raspberrypi.com/documentation/computers/configuration.html#setting-up-a-headless-raspberry-pi where it says "Once an image is created on an SD card, by inserting it into a card reader on a Linux or Windows machines the [boot folder](https://www.raspberrypi.com/documentation/computers/configuration.html#the-boot-folder) can be accessed. Adding certain files to this folder will activate certain setup features on the first boot of the Raspberry Pi.", I was suggesting it might be worth mentioning that RPi-Imager can perform that step directly. :shrug:

**Comment by aallan** [comment]
>  I was suggesting it might be worth mentioning that RPi-Imager can perform that step directly. 🤷

Yup, agree with this. Should mention it.

**Pull request merged**